Makefile

```diff
@@ -22,9 +22,5 @@ httpbin-publish: httpbin-container
 	docker push bluehoodie/httpbin
 
 smoke-test: install
```

Comment:
i would probably call it `test` instead of `smoke-test`.  its cleaner :)

Reply:
done